config.example.json

```diff
@@ -12,14 +12,14 @@
         "symbol": "ETC",
         "name": "Ethereum Classic",
         "title": "Ethereum Classic Block Explorer",
-        "author": "Elaine, Cody, Hackmod, Bakon",
```

Comment:
I think we should ask authors if they are ok with this change
cc @elaineo @realcodywburns @hackmod not sure what handle Bakon has on github

Reply:
Actually, that is not related to copyright changes nor the authors, it is just an additional html tag with the author section, but they are not required by google search engine anymore.